packages/web-frontend/package.json

```diff
@@ -16,12 +16,12 @@
     "build-desktop": "node scripts/build.js && rm -rf builds/desktop && mv build builds/desktop",
     "build-mobile": "REACT_APP_APP_TYPE=mobile node scripts/build.js && rm -rf builds/mobile && mv build builds/mobile",
     "cypress:generate-config": "yarn run:babel cypress/scripts/generateConfig",
-    "cypress:open": "cypress open",
-    "cypress:run": "cypress run",
+    "cypress:open": "cross-env NODE_ENV=test cypress open",
+    "cypress:run": "cross-env NODE_ENV=test cypress run",
     "lint": "eslint --ignore-path ../../.gitignore .",
     "lint:fix": "yarn lint --fix",
     "run:babel": "babel-node --config-file \"../../babel.config.json\"",
-    "start": "PORT=8088 node scripts/start.js",
+    "start": "cross-env PORT=8088 node scripts/start.js",
```

Comment:
Pretty sure that this will be fine, but we should test that a feature branch is successfully deployed after this change